ENCODE_import_data.py

```diff
@@ -147,11 +148,16 @@ def attachment(path):
     filename = os.path.basename(path)
     mime_type, encoding = mimetypes.guess_type(path)
     major, minor = mime_type.split('/')
-    detected_type = magic.from_file(path, mime=True).decode('ascii')
+    try:
+        detected_type = magic.from_file(path, mime=True).decode('ascii')
+    except AttributeError:
+        detected_type = magic.from_file(path, mime=True)
 
     # XXX This validation logic should move server-side.
     if not (detected_type == mime_type or
+            detected_type == 'application/gzip' and mime_type == 'application/x-tar' or
```

Comment:
detected_type is ```application/x-gzip``` in some Macs and Linux environments, I think we want 
```detected_type in ['application/gzip', 'application/x-gzip', 'application/[any_other_compresse_mime_types]'] and mime_type == 'application/x-tar'``` 
in this case